scripts/update_taxonomies.py

```diff
@@ -18,7 +18,7 @@
 logging.basicConfig(level=logging.INFO)
 
 
-def update_taxonomies(filename):
+def update_taxonomies(filename, warn=True):
```

Comment:
Flags like `warn` are unnecessary when using the `logging` module. You can just do 

```python
logger.warn('....')
```

Then expose a way to change the logging level from the CLI:

```python
if '--warn' in sys.argv:
    logger.setLevel(logging.WARNING)
```

(Note: the scripts [README](https://github.com/CenterForOpenScience/osf.io/blob/8019d1510b79619a59b1818a212fe5c8cc311342/scripts/README.md#L40-L57) has a more complete example)